pipelines/build/regeneration/build_job_generator.groovy

```diff
@@ -166,7 +166,10 @@ node ("master") {
 
     def excludes = (params.EXCLUDES_LIST) ?: ""
     def jenkinsCreds = (params.JENKINS_AUTH) ?: ""
-    Integer sleepTime = (params.SLEEP_TIME) != "" ? Integer.parseInteger(SLEEP_TIME) : 900
+    Integer sleepTime = 900
+    if (params.SLEEP_TIME) {
+      sleepTime = SLEEP_TIME as Integer
+    }
```

Comment:
Could you just change the original line to `? SLEEP_TIME.toInteger()` ?

Reply:
I remember trying that before when SLEEP_TIME was first implemented, it produced the same error

Reply:
Odd. We use it in the OpenJ9 scripts.
https://github.com/eclipse/openj9/search?q=toInteger%28%29&type=
Well whatever works. Tomato tomahtoe